suites/apis/requestorNewTest.js

```diff
@@ -40,7 +61,8 @@ Scenario('User requests access for a policy with a signed status (and revoke it
 
   // Create a request to 'revoke' a policy that is present in Arborist again
   // with request status set to SIGNED
-  await requestorTasks.createRequestForPolicyID(accessTokenHeader, users.user0.username, 'requestor_integration_test', true, 'SIGNED');
+  const createResponseSecond = await requestorTasks.createRequestForPolicyID(accessTokenHeader, users.user0.username, 'requestor_integration_test', true, 'SIGNED');
```

Comment:
I feel like I'm probably nit-picking, but maybe you can change the name of this variable to `createRevokeResponse`?